Mathlib/CategoryTheory/Limits/Shapes/WidePullbacks.lean

```diff
@@ -268,72 +309,77 @@ noncomputable abbrev widePushout (B : C) (objs : J → C) (arrows : ∀ j : J, B
   colimit (WidePushoutShape.wideSpan B objs arrows)
 #align category_theory.limits.wide_pushout CategoryTheory.Limits.widePushout
 
-variable (C)
+-- variable (C) -- Porting note: Lean had problems with which `C` below. Changed to `D`
 
 namespace WidePullback
 
-variable {C} {B : C} {objs : J → C} (arrows : ∀ j : J, objs j ⟶ B)
+variable {D} [Category.{v₂} D] {B : D} {objs : J → D} (arrows : ∀ j : J, objs j ⟶ B)
 
 variable [HasWidePullback B objs arrows]
 
 /-- The `j`-th projection from the pullback. -/
 noncomputable abbrev π (j : J) : widePullback _ _ arrows ⟶ objs j :=
   limit.π (WidePullbackShape.wideCospan _ _ _) (Option.some j)
+set_option align.precheck false in
 #align category_theory.limits.wide_pullback.π CategoryTheory.Limits.widePullback.π
 
 /-- The unique map to the base from the pullback. -/
 noncomputable abbrev base : widePullback _ _ arrows ⟶ B :=
   limit.π (WidePullbackShape.wideCospan _ _ _) Option.none
+set_option align.precheck false in
 #align category_theory.limits.wide_pullback.base CategoryTheory.Limits.widePullback.base
 
-@[simp, reassoc.1]
+@[reassoc (attr := simp)]
 theorem π_arrow (j : J) : π arrows j ≫ arrows _ = base arrows := by
-  apply limit.w (wide_pullback_shape.wide_cospan _ _ _) (wide_pullback_shape.hom.term j)
+  apply limit.w (WidePullbackShape.wideCospan _ _ _) (WidePullbackShape.Hom.term j)
+set_option align.precheck false in -- Porting note: it is in the context what is going on?
```

Comment:
I'm not sure what is meant with this porting note.
Also, I see that a lot of these `set_option`s are tagged with a `Porting note: again`, but not all of them.

Reply:
I missed a lower case w. They are all gone now